### ISSUE TITLE
스팸 이미지 필터링  submit 오류(y_pred 데이터 타입)

### ISSUE THREAD

**Issue by developer0hye** [AI-RUSH-Operation/NAVER-AI-RUSH#50]
**Informations**
#### CLI
- **OS (Mac, Windows, Linux, etc) and version**: Windows
- **client version(please show nsml --version)**:
Built: 2020-07-16T14:06:32+09:00
Go: go1.12.17
GOARCH: amd64
GOOS: windows
Revision: abae53dd3589a47d7f89a40128cb86176123f7ec (not-modified)
Preconfigured connection: airush-cli.nsml.navercorp.com:10001

#### WEB
- **Brower (chrome, firfox, etc)**: chrome
- **URL**:

**What is your login ID?**
developer0hye

**Question**

![image](https://user-images.githubusercontent.com/35001605/87655496-4c278600-c793-11ea-80d5-7e6d6331bbbb.png)

Keras 로 작성된 베이스 라인 코드를 PyTorch  기반의 코드로 변경중입니다.

infer 코드까지 모두 새로 작성한 후 submit 을 한 결과, 정답 비교단계에서 예측값과 타겟값의 자료형이 맞지 않는 오류가 발생하였습니다.

![image](https://user-images.githubusercontent.com/35001605/87655818-ac1e2c80-c793-11ea-93fa-8806211d10f7.png)

현재 infer 코드와 관련하여 y_pred 의 경우 원소 값으로 0, 1, 2, 3 (정수)값을 가질 수 있는 numpy array 를 저장하도록 구현하였습니다. y_pred 의 원소로써 저장되는  값(0, 1, 2, 3)을 str형으로도 해보고 아예 클래스("normal", ...) 로도 해보았으나 같은 오류가 계속 출력되었습니다.

y_pred 에  어떠한 데이터가 들어가야하는지 확인할 방법이 있을까요??

테스트 과정에 필요한 디버깅 과정에 어려움을 겪고 있어 이렇게 질문을 드립니다.

**Comment by developer0hye** [comment]
submit -t 로 제출하였을때는 정상 동작합니다.

**Comment by nsml-admin** [comment]
제출하신 값의 형식이 이상한것같습니다.

아래와 같이 값을 리턴하고있는데, y_pred.detach().cpu() 같은 함수를 써서 integer값만 반환하도록 해주시기 바랍니다.

`0,xxxxxxxxxxxxxxxxxxxxxxxxxxx.jpg,"tensor(0, device='cuda:0')"`

**Comment by developer0hye** [comment]
@nsml-admin 
감사합니다. 해결되었습니다!
